overlap-scripts/kubectl-kubestellar-ensure-mbw

```diff
@@ -0,0 +1,152 @@
+#!/usr/bin/env bash
+
+# Copyright 2023 The KubeStellar Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Purpose: ensure that a mailbox workspace has the needed KubeStellar CRDs.
+
+# Usage: $0 ($kubectl_flag | --with-kube boolean | -X)* mbw_name
```

Comment:
The caller of this script, the mailbox controller, does not know whether the user wants the Kubernetes containerized workload management resources defined. That is not even a constant for all time.

Reply:
I think that once we add the space support, we will configure that according to the space provider. i.e., we will have pre-defined function of  `is-kube-resources-bind-needed(provider)  Boolean`

Reply:
We have a design document.  Please review the section (including its subsections) at https://docs.kubestellar.io/main/Coding%20Milestones/PoC2023q1/outline/#workload-management-workspaces-wmw . The design document needs to be updated as the design changes.

As it currently stands now (before switching to the space framework), the set of resources (resource = sort of object, not individual object) that are natured in a WDS is a design-time constant. Since THIS PR is not about switching to to space framework, maybe this aspect does not need to change in this PR?